benches/centralised_repetition.rs

```diff
@@ -0,0 +1,85 @@
+use criterion::{black_box, criterion_group, criterion_main, Criterion};
+use rand_chacha::ChaCha20Rng;
+use rand_core::SeedableRng;
+use std::time::Duration;
+
+use caledonia::centralised::Proof;
+
+pub mod criterion_helpers;
+pub mod utils;
+
+fn prepetitions(
+    c: &mut Criterion<criterion_helpers::Repetitions>,
+    lambdas: &[usize],
+    s_p: &[usize],
+    n_p: &[usize],
+    n_f: &[usize],
+) {
+    let mut group = c.benchmark_group("Centralised".to_string());
+
+    fn prove_repetitions(
```

Comment:
Would be nice to add some high level comment on what this function is doing exactly.